aws/cloudformation/components/database.yml.erb

```diff
@@ -1,6 +1,5 @@
 <%
-  require 'aws-sdk-sts'
-  PRODUCTION_DB_CLUSTER_ID = "arn:aws:rds:us-east-1:#{Aws::STS::Client.new.get_caller_identity.account}:cluster:production-aurora-cluster".freeze
+  PRODUCTION_DB_CLUSTER_ID = "production-aurora-cluster".freeze
```

Comment:
We could use `CDO.db_cluster_id`, but that has a different value on a developer's workstation and makes it harder for them to execute the `rake stack:validate` task locally.